smartcard/pcsc/constants.go

```diff
@@ -2,6 +2,8 @@ package pcsc
 
 const (
     // Scope
+    CARD_SCOPE_USER = 0x0000
```

Comment:
Create aliases in smartcard package. E.g. smartcard.SCOPE_USER, smartcard.SCOPE_SYSTEM

Reply:
Done.